tests/extrusion/test_variable_layers_mesh_volume.py

```diff
@@ -1,4 +1,5 @@
 import numpy
+from pyop2 import mpi
 from firedrake import *
 from firedrake.mesh import _from_cell_list as create_dm
```

Comment:
Should probably have a public wrapper around `_from_cell_list` which takes a public comm.